azurelinuxagent/common/utils/networkutil.py

```diff
@@ -119,12 +119,12 @@ class AddFirewallRules(object):
     Please make sure to not log anything in any function this class.
     """
 
-    #-A adds the rule to the eno of the ip table chain
+   # __ACCEPT_COMMAND adds the rule to the end of the iptable chain
```

Comment:
PEP 8 violation here - 

`PEP 8: E114 indentation is not a multiple of four (comment)`